.ci_support/linux_64_.yaml

```diff
@@ -5,6 +5,12 @@ channel_sources:
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
-target_platform:
-- linux-64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+pin_run_as_build:
+  python:
+    min_pin: x.x
```

Comment:
Introduction and reuse of `python_min` parameter in the `host` and `test` sections of the recipe makes sure that the same Python version is used and expected in both build and test environments preventing the unnecessary build.